source/sessions/snapshot-sessions.rst

```diff
@@ -66,10 +66,10 @@ Snapshot reads
     Snapshots reads are majority committed reads.
 
 Snapshot timestamp
-    Snapshot timestamp, representing timestamp of the first read (find/aggregate/distinct operation) in the session.
+    Snapshot timestamp, representing timestamp of the first supported read operation (i.e. find/aggregate/distinct) in the session.
     The server creates a cursor in response to a snapshot find/aggregate command and 
-    reports ``atClusterTime`` field in the cursor response. For distinct command server adds ``atClusterTime`` field to the distinct response object.
-    ``atClusterTime`` field represents the timestamp of the read and is guaranteed to be majority committed.
+    reports ``atClusterTime`` within the ``cursor`` field in the response. For distinct commands the server adds a top-level ``atClusterTime`` field to the response.
```

Comment:
"for the distinct command the server" to match "find/aggregate command" singular form?

Reply:
```suggestion
    reports ``atClusterTime`` within the ``cursor`` field in the response. For the distinct command the server adds a top-level ``atClusterTime`` field to the response.
```